apps/prairielearn/assets/scripts/tsconfig.json

```diff
@@ -1,6 +1,6 @@
 {
   "compilerOptions": {
-    "lib": ["ES2020", "DOM"],
+    "lib": ["ES2021", "DOM"],
```

Comment:
This change was necessary to make the type `Intl.DateTimeRangeFormatPart` available, as the new version of `@js-temporal/polyfill` relies on it. In 2025, this should be a reasonable lower bound for browser versions.